sycl/test-e2e/syclcompat/memory/usm_allocations.cpp

```diff
@@ -52,24 +51,10 @@ template <typename T> void test_host() {
   USMTest<T> usm_fixture;
   if (usm_fixture.skip)
     return; // Skip unsupported
-
   if (!usm_fixture.q_.get_device().has(sycl::aspect::usm_host_allocations))
     return; // Skip unsupported
-  usm_fixture.d_A = syclcompat::malloc_host<T>(usm_fixture.size_);
-  usm_fixture.launch_kernel();
-  usm_fixture.check_result();
-  syclcompat::free(usm_fixture.d_A);
-}
 
-template <typename T> void test_shared() {
-  std::cout << __PRETTY_FUNCTION__ << std::endl;
-  USMTest<T> usm_fixture;
-  if (usm_fixture.skip)
-    return; // Skip unsupported
-
-  if (!usm_fixture.q_.get_device().has(sycl::aspect::usm_shared_allocations))
-    return; // Skip unsupported
-  usm_fixture.d_A = syclcompat::malloc_shared<T>(usm_fixture.size_);
+  usm_fixture.d_A = syclcompat::malloc_host<T>(usm_fixture.size_);
```

Comment:
does `d_A` not suggest that this is device memory not host?

Reply:
Yes, Thanks for spotting that. Fixed.